### DISCUSSION TITLE
Video freezes when Admob's ad load Flutter

### DISCUSSION THREAD

**Discussion by amit0021** [jhomlala/betterplayer#678] [Q&A]
Hey there everyone  I want to implement Admob ads in my app. Basically my app is about video streaming. Whenever the ads load the audio works fine but there is a lag in video. Can anyone help me with this?

My Code is more or less like this.
https://stackoverflow.com/questions/67800413/video-freezes-when-admobs-ad-load-flutter

**Comment by jhomlala** [comment]
What do you mean with "there is a lag in video". Can you elaborate more or attach some video from your app?
